python/paddle/tensor/manipulation.py

```diff
@@ -4169,6 +4169,11 @@ def scatter(
     **Scatter Layer**
     Output is obtained by updating the input on selected indices based on updates.
 
+    In the following figure, when overwrite is True, update the output of the same index using the overlay mode, in turn updating ``x[index[i]]`` to ``update[i]``; When overwrite is False, the output of the same index is updated using the summative mode, which first updates ``x[index[i]]`` to a Tensor of the same size as the row with values of 0. Add ``update[i]`` to ``x[index[i]]`` in turn to produce output.
```

Comment:
感觉这块儿有点描述不够清晰，`overlay`好像挺少用的，`in turn updating...` 我一开始没看明白
```suggestion
    As shown in the figure,  when ``overwrite`` is set to ``True``, the output for the same index is updated in overwrite mode, where ``x[index[i]]`` is directly replaced with ``update[i]`` sequentially; When ``overwrite`` is set to ``False``, the output for the same index is updated in accumulation mode. In this mode, ``x[index[i]]`` is first initialized with elements set to 0. Then, ``update[i]`` is sequentially added to ``x[index[i]]`` to produce the output.
```

Reply:
应该已经按照您的意见修改好并重新提交啦~